File: javascript/8_kyu/convert_number_to_reversed_array_of_digits.js
// Convert number to reversed array of digits
// https://www.codewars.com/kata/5583090cbe83f4fd8c000051/train/javascript

function digitize(n) {
    var lis = n.toString().split('');
    var ans = [];
    for (var i = 0; i < lis.length; i++) {
      ans.push(parseInt(lis[i]));
    }
    
    return ans.reverse();
  }

File: swift/8_kyu/convert_number_to_reversed_array_of_digits.swift
// Convert number to reversed array of digits
// https://www.codewars.com/kata/5583090cbe83f4fd8c000051/train/swift

func digitize(_ num: Int) -> [Int] {
    let digits = String(num)
    let ans = digits.compactMap{ $0.wholeNumberValue }
    return ans.reversed()
}

print(digitize(23582357))
print(digitize(45762893920))
print(digitize(548702838394))